### ISSUE TITLE
Provide process loading indication

### ISSUE THREAD

**Issue by nimrodne** [warpdotdev/Warp#2329]
### Describe the solution you'd like?

```text
There isn't any indication that there is some process loading in the terminal tab. The native mac terminal for example, has a loading spinner in the tab title, when a process is currently running.
```

### How important is this feature to you?

4

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

None

**Comment by dannyneira** [comment]
Thanks for this feature request! Please :+1:  and subscribe to this request as it helps us gauge interest. We will post any updates here.

**Comment by dannyneira** [comment]
As a workaround, check out the notifications feature, it will notify once a long-running process is complete, or password input is required in a terminal. https://docs.warp.dev/features/notifications

**Comment by xyzzy314** [comment]
A variation on the Feature request here - I use a lot of tabs with remote SSH sessions, and sometimes those sessions may generate output.  At the moment, Warp may just bounce the icon (which I hate), but, there is no way when looking at Warp to determine which of my tabs had something take place.

This is a feature in Terminal.app that I definitely miss.  Would be good to have some way for the tab that's had something take place (when not currently the focussed tab) visually indicate itself.

**Comment by elviskahoro** [comment]
Closing in favor of #2187
